src/printer.js

```diff
@@ -628,7 +628,23 @@ function genericPrintNoParens(path, options, print) {
 
       return concat(parts);
     case "CallExpression": {
-      const parent = path.getParentNode();
+      if (
+        // We want to keep require calls as a unit
+        n.callee.type === "Identifier" && n.callee.name === "require" ||
+        // `it.only('long name', () => {` should not break
+        (n.callee.type === "Identifier" ||
+          (n.callee.type === "MemberExpression" && n.callee.object.type === "Identifier" && n.callee.property.type === "Identifier")) &&
+          n.arguments.length === 2 &&
+          (n.arguments[0].type === "StringLiteral" || n.arguments[0].type === "Literal" && typeof n.arguments[0].value === "string") &&
+          (n.arguments[1].type === "FunctionExpression" || n.arguments[1].type === "ArrowFunctionExpression") &&
+          n.arguments[1].params.length <= 1
```

Comment:
I'm down. I'll add `it` and `test`. If you are using something else, too bad for you, it's going to indent you!